INSTALL

```diff
@@ -0,0 +1,41 @@
+There are two ways to install Johnny which depends on your platform.
+1. Use our easy installer (Windows and OS X only)
+2. Download source and build Johnny (all platforms)
+
+=====================
+Windows and OS X only
+=====================
+
+1. Easy-install : use our installer
+-----------------------------------
+Simply visit http://openwall.info/wiki/john/johnny to get the latest installer
+for your platform and http://openwall.info/wiki/john/custom-builds to get the latest
+John The Ripper binaries.
+
+============================
+Unix/Linux, Windows and OS X
```

Comment:
This isn't exactly what I had in mind...

Can you please take another look at the example I wrote for Tarsnap https://github.com/shinnok/tarsnap-gui/blob/master/INSTALL and try a bit harder to come up with something more along the lines of that one?